babi/screen.py

```diff
@@ -131,12 +131,19 @@ def _draw_header(self) -> None:
             filename += ' *'
         if len(self.files) > 1:
             files = f'[{self.i + 1}/{len(self.files)}] '
-            version_width = len(VERSION_STR) + 2 + len(files)
         else:
             files = ''
-            version_width = len(VERSION_STR) + 2
-        centered = filename.center(self.margin.cols)[version_width:]
-        s = f' {VERSION_STR} {files}{centered}{files}'
+
+        before, after = f' {VERSION_STR} {files}', f'{files}'
```

Comment:
the second fstring here does nothing

Reply:
A question about this:
As far as I understand this, the right hand side files indicator would always be offscreen.
Firstly, am I correct in my understanding of this?
Second, is this intentional, should it be shown, or should it just go away?

Reply:
yeah that's a mistake on my part 😆 -- it should just go away.  that's what you get for writing code while streaming 😆